src/components/Navbar.jsx

```diff
@@ -1,38 +1,67 @@
-export default Navebar = () => {
+import { useEffect, useState } from "react";
+import Image from "next/image";
+
+
+export default function  Navbar () {
+  const [visible, setVisible] = useState(false);
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  useEffect(() => {
```

Comment:
i think this function is not supposed to be here, can you pls specify any particular reasons for not keeping in `page.js` and using it here?

Reply:
which function you want to know about please mention.

Reply:
Useeffect

Reply:
UseEffect is used for the navbar and only in the Navbar.jsx component no need every where.